src/main/app/case/from-api-format.test.ts

```diff
@@ -314,4 +314,15 @@ describe('from-api-format', () => {
       ],
     });
   });
+
+  test('converts empty coClarificationResponses list value to empty string', () => {
+    const nfdivFormat = fromApiFormat({
+      ...results,
+      coClarificationResponses: [],
+    } as unknown as CaseData);
+
+    expect(nfdivFormat).toMatchObject({
+      coClarificationResponses: '',
+    });
+  });
```

Comment:
is it worth adding in a test for when the list isn't empty? And also below for if a response has been entered?